agenda/app/Http/Requests/ChangePassword/UpdatePasswordRequest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Requests\ChangePassword;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Http\Requests\Traits\SanitizesInput;
+
+class UpdatePasswordRequest extends FormRequest
+{
+    use SanitizesInput;
+
+   /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     *  Filters to be applied to the input.
+     *
+     *  @return array
+     */
+    public function filters()
+    {
+        return [
+            'after' => [
+                'current_password'      => 'cast:string',
+                'new_password'          => 'cast:string',
+                'confirm_new_password'  => 'cast:string',
+                'token_update_password' => 'cast:string',
+            ],
+        ];
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'current_password'      => 'required|string|',
+            'new_password'          => 'required|string|regex:/^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z]).{8,20}$/',
+            'confirm_new_password'  => 'required|string|same:new_password',
+            'token_update_password' => 'required|string|',
```

Comment:
Acho que o token não precisa ser informado aqui, pois a request vai bater no middleware primeiro... poderia testar fazendo favor? Se for isso mesmo, vc pode verificar se o token_update_password é null lá no middleware, e se for pode retornar o erro com o método sendError da trait e colocar até o status code 422